src/locales/pl.json

```diff
@@ -0,0 +1,549 @@
+{
+  "info": {
+    "code": "pl",
+    "lang": "Polski",
+    "lang-english": "Polish",
+    "author": "Nicke Manarin",
```

Comment:
Hi, thank you for your work!
Would you like to add your name or nickname as the author of the translation?

Reply:
Yes, you can add my nickname "purple black". Thanks!